cfme/tests/containers/test_containers_properties.py

```diff
@@ -1,4 +1,5 @@
 # -*- coding: utf-8 -*-
+from random import sample
```

Comment:
Bit of housekeeping, drop this below `import pytest` and add in an empty newline below it, before `from cfme.(...)`